templates/hound.yml

```diff
@@ -6,12 +6,12 @@ haml:
   # config_file: .haml-style.yml
   enabled: true
 javascript:
-  # config_file: .javascript-style.json
+  config_file: .jscsrc
```

Comment:
I just realized my markdown was janky and the link was broken. [Fixed](http://eslint.org/blog/2016/04/welcoming-jscs-to-eslint).

They didn't give any concrete timeline, but in an effort to push people away from it, JSCS has already been [unlisted](https://github.com/houndci/hound/pull/1129) from Hound's documentation.

> We expect this effort to take several months, and as such, we still encourage current JSCS users to upgrade to JSCS 3.0.0 and provide feedback to the team. We’ll announce when all of the changes necessary to support JSCS users in ESLint are complete and will start encouraging JSCS users to switch to ESLint at that time.